Sources/Widgets/Widgets3D/ResliceCursorWidget/behavior.js

```diff
@@ -68,6 +68,50 @@ export default function widgetBehavior(publicAPI, model) {
     model.widgetState.deactivate();
   };
 
+  publicAPI.handleStartMouseWheel = (callData) => {
+    publicAPI.invokeStartInteractionEvent();
+    // When interacting, plane actor and lines must be re-rendered on other views
+    publicAPI.getViewWidgets().forEach((viewWidget) => {
+      viewWidget.getInteractor().requestAnimation(publicAPI);
+    });
+  };
+
+  publicAPI.handleMouseWheel = (calldata) => {
+    const direction = calldata.spinY;
+    const currentCenter = model.widgetState.getCenter();
+    const imageSpacing = model.widgetState.getImage().getSpacing();
+
+    // Get the normal use to translate the center along
+    const normal = calldata.pokedRenderer
+      .getRenderWindow()
+      .getRenderers()[0]
+      .getActiveCamera()
+      .getDirectionOfProjection();
+
+    // Direction of the projection if the inverse of what we want
+    const translationVector = vtkMath.multiplyScalar(normal, -1);
+
+    const newCenter = [
+      currentCenter[0] + direction * translationVector[0] * imageSpacing[0],
+      currentCenter[1] + direction * translationVector[1] * imageSpacing[1],
+      currentCenter[2] + direction * translationVector[2] * imageSpacing[2],
+    ];
+
+    model.widgetState.setCenter(newCenter);
```

Comment:
isn't it going to go outside the bounds ?

Reply:
I thaught you didn't want to check where the center is ?

Reply:
I said:  _you SHOULD PROJECT new center onto the edge if it is outside_